doc/nrf/libraries/networking/downloader.rst

```diff
@@ -187,12 +187,30 @@ The following snippet shows how to download a file using HTTPS:
 Limitations
 ***********
 
-The library requires the host server to provide a Content-Range field in the HTTP GET response header when using HTTPS with the nRF91 Series devices.
-If this header field is missing, the library logs the following error::
+The following limitations apply to this library:
 
-   <err> downloader: Server did not send "Content-Range" in response
+nRF91 Series TLS limitation
+===========================
 
- Due to internal limitations, maximum CoAP block size is 512 bytes.
+The nRF91 Series modem has a size limit of ~2 kB for receiving TLS packages.
+The library therefore asks the server for a content-range which must be supported by the host server when using HTTPS with the nRF91 Series devices.
+
+The content range is set by the :c:member:`download_client_cfg.frag_size_override` configuration in the download client configuration.
```

Comment:
Is this the correct data type for this library?

Reply:
No, it was copied from the closed known issue PR, will update, thanks!